spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "2.1"
+services:
+  spark:
+    image: feathub-spark:latest
+    container_name: spark
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    environment:
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+    volumes:
+      - ./main.py:/tmp/main.py
+      - ./data:/tmp/spark-data
+    command:
+      - bash
+      - -c
+      - "./sbin/start-master.sh && ./sbin/start-worker.sh spark://spark:7077; while true; do sleep 10; done"
```

Comment:
Would it be simpler to use `./sbin/start-all.sh`?

Reply:
I'm afraid not, `start-all.sh` needs some `ssh` setup, which is more complex.